### ISSUE TITLE
Not really an issue but a question...

### ISSUE THREAD

**Issue by h8uthemost** [Turro75/MyMinUI#18]
I have an og rg35xx, running a 2 card setup with MinUI. TF1(16gb card)has all the system files and TF2(128gb card) has all my roms and bios. 

So my question is, is it possible to simply "update" my TF1 to Myminui? Or am I going to have to wipe it and do what the instructions say, which is flash anbernic again then install Myminui.  Also, my TF2 card should remain as is right? I don't need to insert it while installing Myminui on the TF1 card, or am I wrong about that?

Basically I just don't want to touch my 128gb TF2 card that has all my roms and bios. It would take forever to copy all of that to my computer to backup and then move it all back onto the card after the install. The card has alot on it. 

Is this possible or should I not touch anything and stay with MinUI? 

Thanks for any help. And great work with what you're doing. I'm using Myminui on my a30 and am loving it.
